### PR TITLE
[16.0][MIG] users_ldap_mail: Migration to version 16.0

### PR DESCRIPTION
**Pull request by Yadier-Tecnativa**
@Tecnativa TT43251

### CODE REVIEW

**Comment by Yadier-Tecnativa** [comment]
ping @sergio-teruel , @pedrobaeza

File: users_ldap_mail/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright Daniel Reis (https://launchpad.com/~dreis-pt)
+# Copyright 2021 Tecnativa - João Marques
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "LDAP mapping for user name and e-mail",
+    "version": "16.0.1.0.1",
```
Comment (pedrobaeza) [inline users_ldap_mail/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Review by pedrobaeza — COMMENTED** [review]
And please squash a bit administrative commits:

https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

Have you checked that the involved methods still exists upstream as there are no tests in the module?

**Comment by Yadier-Tecnativa** [comment]
Done, please review @carlosdauden , @sergio-teruel

**Review by pedrobaeza — APPROVED** [review]
Basic migration, so we can merge:

/ocabot merge nobump

**Pull request merged**